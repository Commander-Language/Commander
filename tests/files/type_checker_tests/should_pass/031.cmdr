//Test bindings type check
x = fn () -> println "hi world";
y = x;
