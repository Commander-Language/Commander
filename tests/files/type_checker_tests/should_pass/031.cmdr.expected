(PrgmNode
	(ExprStmtNode (BinOpExprNode (BindingNode x) SET (LambdaExprNode (BindingsNode) (PrintlnStmtNode (StringExpressionNode (StringNode (StringExprsNode (StringExpressionNode (StringNode 'hi world') STRING))) STRING)) (fn () -> VOID)) (fn () -> VOID)))
	(ExprStmtNode (BinOpExprNode (BindingNode y) SET (LValueExprNode (VarExprNode x (fn () -> VOID)) (fn () -> VOID)) (fn () -> VOID))))
